tap_shopify/streams/base.py

```diff
@@ -21,7 +22,10 @@
 
 def is_not_status_code_fn(status_code):
     def gen_fn(exc):
-        return exc.code not in status_code
+        if getattr(exc, 'code', None):
```

Comment:
I would only quibble over style here. I think we'd want two levels of `in` calls.

```
(orchestrator) vagrant@taps-tvisher1:/opt/code/orchestrator$ ipython
Python 3.5.2 (default, Nov 23 2017, 16:37:01)
Type 'copyright', 'credits' or 'license' for more information
IPython 6.5.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: r={'code': 500}

In [2]: status_code={500}

In [3]: 'code' in r and r['code'] not in status_code
Out[3]: False

In [4]: status_code={400}

In [5]: 'code' in r and r['code'] not in status_code
Out[5]: True

In [6]: r={}

In [7]: 'code' in r and r['code'] not in status_code
Out[7]: False
```